conferences/2020/general.json

```diff
@@ -1047,6 +1058,17 @@
     "cfpUrl": "https://thestrangeloop.com/cfp.html",
     "cfpEndDate": "2020-04-28"
   },
+  {
+    "name": "NDC Sydney",
+    "url": "https://ndcsydney.com/",
+    "startDate": "2020-10-12",
+    "endDate": "2020-10-16",
+    "city": "Sydney",
+    "country": "Australia",
+    "twitter": "@NDC_Conferences",
+    "cfpUrl": "hhttps://ndcsydney.com/page/call-for-papers/",
```

Comment:
@philnash can you please delete an extra `h` in the cfpUrl? Thanks!

Reply:
Bah, that was dumb! Thanks @cgrail for fixing that up for me.